Documentation/Signoffs/ResistanceSystem.md

```diff
@@ -0,0 +1,73 @@
+Team 4 – Mario Kart & Ride Replay Simulator
+Team Members: Ray Durlin, Blake Pickett, Tyler Chittum, Benjamin Reed, and Sage Mooneyham
+Detail Design: Resistance System
+
+
+Function of the Subsystem: 
```

Comment:
Use markdown headings by starting the line with a pound sign. Add additional pound signs to reduce the size of the header.